lib/pages/pool/MyLiqudity/MyLiquidity.dart

```diff
@@ -249,7 +250,7 @@ class _MyLiquidityState extends State<MyLiquidity> {
               child: Container(
                 child: TextFormField(
                   onChanged: (value) {
-                    setState(() {});
+                    bloc.add(SearchBarInputEvent(searchBarInput: value));
```

Comment:
Needs a check here, trim the input or check if the input contains characters. Just good practice. Otherwise, it shows an empty list on whitespaces.